### DISCUSSION TITLE
kurtosis software request

### DISCUSSION THREAD

**Discussion by ihattan2020** [neurodesk/neurodesk.github.io-hugo#462] [General]
Hi Steff.,

I tried to create a pull-up request, but it didn't allow me to do so. So, I have to be a collaborator to do it. I have the following software; could you please add it?  at Neurodesk CAI-UQ WKS.

https://cai2r.net/resources/diffusion-kurtosis-imaging-matlab-toolbox/
https://pydesigner.readthedocs.io/en/latest/index.html
https://www.nitrc.org/projects/dke/

I don't have the exact versions or the code file, but most importantly is the pydesigner one. I think all the info you need would be in the link above.
I merged this topic into the link below for your information.

https://github.com/NeuroDesk/neurocommand/commit/b664c37ea1cfab121b976a0175e4b5dcc80d05a8

Many thanks in advance,

Ibrahim Hattan

**Comment by stebo85** [comment]
Dear @ihattan2020,

You don't need any permissions to send a pull request.

The first one is just a matlab script. There is nothing we can do there to help. You need to download it and run it in Matlab.

The second and thrid toolbox should be possible to be added.

Kind regards
Steffen

**Reply by stebo85** [reply]
Pydesigner comes as a docker container: https://hub.docker.com/r/dmri/neurodock/tags

**Reply by stebo85** [reply]
@ihattan2020, dke is not maintained anymore:
"Hi DKE users, We now have source code available on GitHub at https://github.com/m-ama We are no longer developing DKE. Instead, we are switching to NYU’s DESIGNER pipeline..."

**Comment by stebo85** [comment]
@ihattan2020 

https://cai2r.net/resources/diffusion-kurtosis-imaging-matlab-toolbox/
Not possible to add to Neurodesk because it's just a matlab script: Simply download and run in Matlab

https://pydesigner.readthedocs.io/en/latest/index.html
Pydesigner is building here: https://github.com/NeuroDesk/neurocontainers/actions/runs/11965487502 and should be available next week for testing

https://www.nitrc.org/projects/dke/
DKE is not possible to add because it is not maintained anymore and requires an outdated Matlab runtime environment

**Reply by ihattan2020** [reply]
Many thanks for your help. at least Pydesiner, please.

**Comment by stebo85** [comment]
Dear @ihattan2020 - this is completed now. pydesigner is available on neurodesk under the tool "neurodock". Can you open a CAI helpdesk ticket to update neurodesk on the workstations and then test pydesigner? Thank you.
